lib/pay/paddle/subscription.rb

```diff
@@ -8,6 +8,7 @@ module Subscription
 
         store_accessor :data, :paddle_update_url
         store_accessor :data, :paddle_cancel_url
+        store_accessor :data, :paddle_paused_from
```

Comment:
Thanks for the change. I was just thinking that in v3 when I reorganize things, we can delegate to the a Paddle::Subscription class and let it handle pausing and that way the logic doesn't have to bleed into the other providers. 👍